tools/scripts/upgrades/increase-block-size.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo "Current params:"
+poktrolld query consensus params
+
+echo "Submitting transaction to change the block size..."
+poktrolld tx authz exec tools/scripts/params/consensus_increase_block_size.json --from pnf --yes
+
+sleep 3
```

Comment:
Is there a way to query the block height and just repeat until it's what we expect it to be?

Reply:
Honestly, I don't think this should be in a script but rather documented.

Reply:
Can you #PUC

1. Why it's 3 seconds? E.g.: that's how long a block is and we planned the upgrade for the next block? I don't know if this is accurate, which is why I'm asking.

2. Maybe add a TODO to wait on new block evens?